bikespace_dashboard/js/app.js

```diff
@@ -1,20 +1,19 @@
-import { SharedState, Component } from './components/main.js';
-import { Map } from './components/map.js'
-import { SummaryBox } from './components/sidebar/summary_box.js';
-import { IssueChart } from './components/sidebar/issue_chart.js';
-import { DayChart } from './components/sidebar/day_chart.js';
-import { DurationTimeOfDayChart } from './components/sidebar/duration_tod_chart.js';
-
+import {SharedState, Component} from './components/main.js';
+import {Map} from './components/map.js';
+import {SummaryBox} from './components/sidebar/summary_box.js';
+import {IssueChart} from './components/sidebar/issue_chart.js';
+import {DayChart} from './components/sidebar/day_chart.js';
+import {DurationTimeOfDayChart} from './components/sidebar/duration_tod_chart.js';
 
 $.ajax({
   url: 'https://api-dev.bikespace.ca/api/v2/submissions?limit=5000',
-  success: function (data, textStatus, jqXHR) {
-      let shared_state = new SharedState(data);
-      let sidebar = new Component("body", "sidebar", shared_state);
-      let summary_box = new SummaryBox("#sidebar", "summary-box", shared_state);
-      let issue_chart = new IssueChart("#sidebar", "issue-chart", shared_state);
-      let day_chart = new DayChart("#sidebar", "day-chart", shared_state);
-      let duration_tod_chart = new DurationTimeOfDayChart("#sidebar", "duration-tod-chart", shared_state);
-      let map = new Map("body", "map", shared_state);
-  }
-});
\ No newline at end of file
+  success: function (data) {
+    const shared_state = new SharedState(data);
```

Comment:
Need to double check this doesn't cause any unintended behaviour

Reply:
`shared_state` holds a constant reference to a SharedState instance and is never reassigned. so we can safely use `const` here